crates/bdk/src/wallet/tx_builder.rs

```diff
@@ -29,7 +29,7 @@
 //!
 //! tx_builder
 //!     // Create a transaction with one output to `to_address` of 50_000 satoshi
-//!     .add_recipient(to_address.script_pubkey(), 50_000)
+//!     .add_recipient(to_address.script_pubkey(), bitcoin::Amount::from_sat(50_000))
```

Comment:
Already imported in line 18

```suggestion
//!     .add_recipient(to_address.script_pubkey(), Amount::from_sat(50_000))
```